dvc/utils/__init__.py

```diff
@@ -357,7 +357,7 @@ def resolve_output(inp, out):
     return ret
 
 
-def resolve_paths(repo, out):
+def resolve_paths(repo, out, always_local=False):
```

Comment:
IIUC, `always_local` is a high-level concept. Here, it does not convey what it does properly: using the base path as an output name. But, I don't have a good name to suggest either (that is succinct), so I'm fine with it as well.  😄

Reply:
I also couldn't think of a better one 🤔